### DISCUSSION TITLE
Question: what does Linus Torvalds think of GitButler...?

### DISCUSSION THREAD

**Discussion by ZaLiTHkA** [gitbutlerapp/gitbutler#4660] [General]
> **disclaimer**: this is all pure curiosity on my part, nothing more..

I first learnt about Git 15-odd years back, and I've been using it almost daily for the last 10-odd years. as a result, I've used a great many Git GUI clients plus a decent number of Git CLI helper tools along the way.. _I_ think it's brilliant how a completely fresh new concept like this can still become "a real thing" in an environment that is already so diverse. 🍻 

but I honestly can't help but wonder, what does the inventor of Git think about GitButler's "virtual branches" concept..? can't imagine that Linus Torvalds hasn't heard of GitButler yet, but I can't seem to find anything regarding his opinion of it through the general search engines..

**Comment by mtsgrd** [comment]
Tbh, I would also love to know! @torvalds

**Comment by schacon** [comment]
I'm fairly certain Linus has not heard of GitButler and would not much care. :)
